src/workerd/server/workerd.capnp

```diff
@@ -76,6 +76,34 @@ struct Config {
   extensions @3 :List(Extension);
   # Extensions provide capabilities to all workers. Extensions are usually prepared separately
   # and are late-linked with the app using this config field.
+
+  alarmScheduler @4 :AlarmConfig;
```

Comment:
Hmm, do we really want this to be top-level config?

It feels confusing that I could configure a durable object namespace that stores to disk, but get in-memory-only alarms. Vice-versa is even more confusing I think.

Also, this means that if someone is serving multiple DO namespaces from the same workerd instance, but later decides that they want to split them up, they can't really do that -- they'll lose their alarms.

Could we instead create a separate alarms database per DO namespace? If the namespace is in-memory, use in-memory handling. If the DO namespace is on-disk, then we store the alarms on-disk in the same directory. I'd call the file something like `metadata.sqlite` or `namespace.sqlite` so that if we find there's per-namespace stuff other than alarms we want to put in there in the future, we can.